runtime/altair/src/lib.rs

```diff
@@ -926,6 +911,16 @@ impl orml_tokens::Config for Runtime {
 	type DustRemovalWhitelist = frame_support::traits::Nothing;
 }
 
+impl orml_asset_registry::Config for Runtime {
```

Comment:
We are kind of in a rush to test this in Altair so that we can then release XCM on Centrifuge for the Pools launch. Fwiw, I am fairly confident given all the tests covering this new feature and how it plays out with our XCM setup.

I understand your concern tho! Could we maybe merge this into a dedicated branch and test run this in test environments before moving forward with deploying it to Altair in Kusama?

Another downside of dropping this from Altair is that all the XCM tests are wired to Altair since the XCM config and para ids are so intertwined in XCM.

@offerijns what's your opinion here?